assets/js/components/HostDetails/HostSettingsPage.jsx

```diff
@@ -31,10 +39,10 @@ function HostSettingsPage() {
   } = useSelector(getCatalog());
 
   const saving = useSelector(isSaving(TARGET_HOST, hostID));
-  const hostChecksExecutionEnabled = !canStartExecution(
-    hostSelectedChecks,
-    saving
-  );
+
+  useEffect(() => {
```

Comment:
Just curious, is this `useEffect()` because `selectedChecks` doesn't arrive synchronously from the selector?

Reply:
Let me double check if I haven't introduced a bug :sweat_smile:

Reply:
Oh I don't think there's a bug at least, just trying to understand the introduction of the `useEffect()` here :sweat_smile:

Reply:
And for completeness that is just what we had in `HostChecksSelection`
```
const [selection, setSelection] = useState([]);
  useEffect(() => {
    setSelection(savedHostSelection);
  }, [savedHostSelection]);
```

Reply:
Alright, fixed here 37c347c6517053c67ab341633e206bce983d61f1